docs/README.md

```diff
@@ -23,6 +23,8 @@ These documents provide you with more architectural details:
 - [Database scheme](./architecture/db-scheme.md)
 - [Authentication and authorization](./architecture/authentication-and-authorization.md)
 - [Integration Systems](./architecture/integration-systems.md)
+- [Proposal: Input validation](./architecture/input-validation.md)
+  - [Proposal: API specifications support](./architecture/api-specs-support.md)
```

Comment:
```suggestion
 - [Proposal: API specifications support](./architecture/api-specs-support.md)
```

Reply:
It was supposed to be a part of the input validation hence the sublist.